.github/CODEOWNERS

```diff
@@ -0,0 +1,157 @@
+### Copied from CODEOWNERS_manual ###
+.github/* @autowarefoundation/autoware-maintainers
+
+### Automatically generated from package.xml ###
+common/autoware_ad_api_msgs isamu.takagi@tier4.jp
+common/autoware_auto_common opensource@apex.ai
```

Comment:
@xmfcx @BonoloAWF By the way, does AWF have a common email like `opensource@apex.ai`?